u-boot/cpu/mips/ar7240/qca_dram.c

```diff
@@ -624,6 +629,11 @@ static inline void qca_dram_set_ddr_cfg2(u32 mem_cas,
 
 	/* Gate open latency = 2 * MEM_CAS */
 	tmp = 2 * mem_cas;
+#if (SOC_TYPE & QCA_AR933X_SOC)
```

Comment:
I just checked datasheets for other QC/A9xxx WiSoCs and it looks that only AR9331 has in its datasheet information that GATE_OPEN_LATENCY should be CAS * 2. Similar information is only in AR7240 datasheet. So, maybe it's just another copy&paste mistake from QC/A... ;)

Thanks for tests, I will let you know about results of my tests.